.github/pull_request_template.md

```diff
@@ -1,10 +1,25 @@
-### What does this PR do?
-- 
+Call the add new employee with http://localhost:3000/dashboard/history endpoint.
```

Comment:
add new employee ?

Reply:
that is temporary until the nav bar is ready